exporter/observiqexporter/config.go

```diff
@@ -60,3 +64,35 @@ func (c *Config) validateConfig() error {
 
 	return nil
 }
+
+// Default agent name will be the hostname
+func defaultAgentName() string {
+	const fallbackAgentName = "otel collector"
+
+	if hn, err := os.Hostname(); err == nil {
+		return hn
+	}
```

Comment:
Nit: Typically you should check `err != nil`.

The reason is that this pattern allows you to bail early from methods as soon as you know something is wrong. This is a repeatable (chainable) pattern. Go is very idiomatic, which makes code very readable for other developers. Although this is a nit, it's worthwhile respecting idioms wherever possible.

Reply:
Will keep this in mind going forward. This instance has been fixed now.